### PR TITLE
Migrate landscape level 1 components

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: src/pages/search/drawer/landscape/Forest.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import Accordion from "pages/search/Accordion";
+import ForestIntegrity from "pages/search/drawer/landscape/forest/ForestIntegrity";
+import ForestLossPersistence from "pages/search/drawer/landscape/forest/ForestLossPersistence";
+
+interface Props {
+  handleAccordionChange: () => {};
```
Comment (erikasv) [inline src/pages/search/drawer/landscape/Forest.tsx:6]: Type this function according to the function definition and usage.

File: src/pages/search/drawer/landscape/HumanFootprint.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import AddIcon from "@mui/icons-material/Add";
+
+import Accordion from "pages/search/Accordion";
+import CurrentFootprint from "pages/search/drawer/landscape/humanFootprint/CurrentFootprint";
+import PersistenceFooprint from "pages/search/drawer/landscape/humanFootprint/PersistenceFootprint";
+import TimelineFootprint from "pages/search/drawer/landscape/humanFootprint/TimelineFootprint";
+
+interface Props {
+  handleAccordionChange: () => {};
```
Comment (erikasv) [inline src/pages/search/drawer/landscape/HumanFootprint.tsx:9]: Same as in Forest

File: src/pages/search/drawer/landscape/PAConnectivity.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import Accordion from "pages/search/Accordion";
+import CurrentPAConnectivity from "pages/search/drawer/landscape/connectivity/CurrentPAConnectivity";
+import TimelinePAConnectivity from "pages/search/drawer/landscape/connectivity/TimelinePAConnectivity";
+import CurrentSEPAConnectivity from "pages/search/drawer/landscape/connectivity/CurrentSEPAConnectivity";
+
+interface Props {
+  handleAccordionChange: () => {};
```
Comment (erikasv) [inline src/pages/search/drawer/landscape/PAConnectivity.tsx:7]: Same as in Forest

**Review by erikasv — CHANGES_REQUESTED** [review]

File: src/pages/search/drawer/landscape/Forest.tsx (hunk 1 above)
Comment (cazapatamar) [reply src/pages/search/drawer/landscape/Forest.tsx]: Done!

File: src/pages/search/drawer/landscape/HumanFootprint.tsx (hunk 2 above)
Comment (cazapatamar) [reply src/pages/search/drawer/landscape/HumanFootprint.tsx]: Done!

File: src/pages/search/drawer/landscape/PAConnectivity.tsx (hunk 3 above)
Comment (cazapatamar) [reply src/pages/search/drawer/landscape/PAConnectivity.tsx]: Done!

File: src/pages/search/drawer/landscape/Forest.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import Accordion from "pages/search/Accordion";
+import ForestIntegrity from "pages/search/drawer/landscape/forest/ForestIntegrity";
+import ForestLossPersistence from "pages/search/drawer/landscape/forest/ForestLossPersistence";
+
+interface Props {
+  handleAccordionChange(level: string, tabLayerId: string): void;
```
Comment (erikasv) [inline src/pages/search/drawer/landscape/Forest.tsx:6]: The definition is fine, but we've been using the other syntax for functions, see [this line](https://github.com/PEM-Humboldt/biotablero-frontend/blob/develop/src/pages/search/drawer/strategicEcosystems/EcosystemsBox.tsx#L14) for example.

**Review by erikasv — CHANGES_REQUESTED** [review]

File: src/pages/search/drawer/landscape/Forest.tsx (hunk 4 above)
Comment (cazapatamar) [reply src/pages/search/drawer/landscape/Forest.tsx]: Done!

**Review by erikasv — APPROVED** [review]

**Pull request merged**